src/components/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { Sidebar } from "./header-components/Sidebar";
+import PropTypes from "prop-types";
+
+export const Header = ({ sortOnClick }) => {
+  const [isActive, setActive] = useState("unclicked");
+  // Makes a one-time toggle. To turn of animations when the user has noticed button.
+  const ToggleClass = () => {
+    if (isActive === "unclicked") {
+      setActive(!isActive);
+    }
+  };
+
+  return (
+    <div className="header">
+      <Sidebar />
+      <div className="title-and-button">
+        <h1>Moon Music</h1>
+        <label className="switch">
+          <input
+            type="checkbox"
```

Comment:
We like that this is a checkbox and you where able to make it look so fun and alive :)
//Izabel & Linda